fdbserver/worker.actor.cpp

```diff
@@ -554,21 +560,27 @@ ACTOR Future<Void> registrationClient(Reference<AsyncVar<Optional<ClusterControl
 			}
 		}
 
-		Future<RegisterWorkerReply> registrationReply =
+		state Future<RegisterWorkerReply> registrationReply =
 		    ccInterface->get().present()
 		        ? brokenPromiseToNever(ccInterface->get().get().registerWorker.getReply(request))
 		        : Never();
-		choose {
+		state double startTime = now();
+		loop choose {
 			when(RegisterWorkerReply reply = wait(registrationReply)) {
 				processClass = reply.processClass;
 				asyncPriorityInfo->set(reply.priorityInfo);
+				TraceEvent("WorkerJoiningCluster").detail("CCID", ccInterface->get().get().id());
```

Comment:
I think would be nice to print the network address of CC as well. 

Maybe rename it to `WorkerRegisterRequest` so that it maps to the name of the request type here which would be easier to remember when debugging.